### PR TITLE
517 Notifications Endpoint

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

**Review by matthopson — APPROVED** [review]

File: web-client/src/presenter/computeds/formattedWorkQueue.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ export const formattedWorkQueue = get => {
 
   workQueue = _.orderBy(workQueue, 'currentMessage.createdAt', 'desc');
 
-  workQueue.unreadCount = workQueue.filter(item => !item.readAt).length;
+  // workQueue.unreadCount = workQueue.filter(item => !item.readAt).length;
```
Comment (kkoskelin) [inline web-client/src/presenter/computeds/formattedWorkQueue.js]: are we able to remove this now?

File: web-client/src/views/Header.jsx [hunk 2]
```diff
@@ -10,11 +10,18 @@ import close from '../../node_modules/uswds/dist/img/close.svg';
...
     toggleMobileMenuSequence: sequences.toggleMobileMenuSequence,
     user: state.user,
   },
-  ({ user, mobileMenu, toggleMobileMenuSequence, signOutSequence }) => {
+  ({
+    user,
+    mobileMenu,
+    toggleMobileMenuSequence,
+    signOutSequence,
+    notifications,
```
Comment (kkoskelin) [inline web-client/src/views/Header.jsx]: alphasort?

**Review by kkoskelin — APPROVED** [review]

File: efcms-service/serverless.yml [hunk 3]
```diff
@@ -520,6 +520,18 @@ functions:
             authorizerId:
               Ref: ApiGatewayAuthorizer
 
+  getNotifications:
```
Comment (codyseibert) [inline efcms-service/serverless.yml:523]: remember to update the swagger.json

Comment (ericsorenson) [reply efcms-service/serverless.yml:523]: updated!

**Pull request merged**